### ISSUE TITLE
index of man pages should be deleted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3053]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3053|
|Type|enhancement|
|Created|2008-02-22 19:28:19Z|
|Branch|https://github.com/twisted/twisted/tree/delete-index-3053|

It's severely broken, and we don't have what's needed yet in Lore to generate it. Let's delete it for next release.

<details><summary>Searchable metadata</summary>

```
trac-id__3053 3053
type__enhancement enhancement
reporter__therve therve
priority__highest highest
milestone__twisted_8_0 twisted-8.0
branch__branches_delete_index_3053 branches/delete-index-3053
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1203708499000000 1203708499000000
changetime__1204310270000000 1204310270000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve commented|
|-|-|

(In [22691]) Branching to 'delete-index-3053'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve removed owner|
|-|-|

That was easy, only the core man pages had an index. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Not disagreeing with the ticket description, but the website currently links to the man index.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve commented|
|-|-|

Well, the website links to http://twistedmatrix.com/projects/core/documentation/man/, not to http://twistedmatrix.com/projects/core/documentation/man/index.html. If we remove the index, we'll still have file listing like here: http://twistedmatrix.com/projects/conch/documentation/man/

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_5"></a>|@radix set owner to @therve|
|-|-|

+1 please merge

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve set status to `closed`|
|-|-|

(In [22729]) Merge delete-index-3053

Author: therve
Reviewer: radix
Fixes [#3053](https://github.com/twisted/twisted/issues/3053)

Remove the index for man pages: it's outdated, and lore isn't ready to
generate a clean one based on the content of the directory.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
